### ISSUE TITLE
Feature Request: Increase beyond 5 rows shown per page

### ISSUE THREAD

**Issue by LankyPoet** [ArtVentureX/sd-webui-agent-scheduler#160]
Would love the option to show more rows per page as I tend to juggle a LOT of jobs at once. Only seeing 5 at a time sometimes leads to a lot of page-switching.

Would love 5,10,15,20, ALL display choices (or similar).

**Comment by artventuredev** [comment]
Hey @LankyPoet, good news here. You can you change the page size in the settings page.

**Comment by LankyPoet** [comment]
Awesome. Thank you very much!

**Comment by LankyPoet** [comment]
I closed this too soon! I enabled the option, but it doesn't actually change the number of rows shown (I selected 50, then applied, shutdown, and restarted SD.NEXT).
I tried this both as a separate tab, and as appending Agent Scheduler to the bottom of the main UI. I did a full shut down and restart each time to see if would enable in one spot vs. another, but it doesn't seem to actually modify things for me.
